python-frontend/src/main/java/org/sonar/python/semantic/SymbolTableBuilder.java

```diff
@@ -131,6 +131,7 @@ public void visitFileInput(FileInput fileInput) {
     fileInput.accept(new FirstPhaseVisitor());
     fileInput.accept(new SecondPhaseVisitor());
     addSymbolsToTree((FileInputImpl) fileInput);
+    handleClassMemberAccess(fileInput);
```

Comment:
Why not define a `ThirdPhaseVisitor` (or any better name) and write `fileInput.accept(new ThirdPhaseVisitor())` to for consistency?